mappings/net/minecraft/state/property/Property.mapping

```diff
@@ -1,8 +1,8 @@
 CLASS net/minecraft/class_2769 net/minecraft/state/property/Property
 	METHOD method_11898 getValues ()Ljava/util/Collection;
 	METHOD method_11899 getName ()Ljava/lang/String;
-	METHOD method_11900 getValue (Ljava/lang/String;)Ljava/util/Optional;
+	METHOD method_11900 parse (Ljava/lang/String;)Ljava/util/Optional;
 		ARG 1 name
-	METHOD method_11901 getName (Ljava/lang/Comparable;)Ljava/lang/String;
+	METHOD method_11901 name (Ljava/lang/Comparable;)Ljava/lang/String;
```

Comment:
Why `name`? I think `getName` sounds better (for example `Properties.FACING.getName(Direction.UP)`).

Reply:
It is naming the value, hence just the verb `name`. It would be confusing with the method that gets the name for property itself, e.g. `Properties.FACING.getName()` may return `"facing"` while `Properties.FACING.getName(Direction.UP)` may return `"up"`

Reply:
Oh, I didn't see that there was another `getName()`.

Reply:
Yeah, that is what we have now. 
![image](https://user-images.githubusercontent.com/7806504/65382993-f483cf80-dcd3-11e9-979a-738289cb8f5f.png)

Reply:
In that case something like `getValueName` would sound better. It's not naming the value, it's getting the value's existing name.

Also applies to

Reply:
Nah, if you have two properties of the same type, they can actually have different names for values as the name-value mapping is a part of the property.